src/Mono.Android/Xamarin.Android.Net/AndroidClientHandler.cs

```diff
@@ -220,6 +220,20 @@ string EncodeUrl (Uri url)
 		}
 
 		/// <summary>
+		/// Returns a custom host name verifier for a HTTPS connection. By default it returns <c>null</c> and
+		/// thus the connection uses whatever host name verification mechanism the operating system defaults to.
+		/// Override in your class to define custom host name verification behavior. The overriding class should
+		/// not set the <see cref="m:HttpsURLConnection.HostnameVerifier"/> property directly on the passed
+		/// <paramref name="connection"/>
+		/// </summary>
+		/// <returns>Instance of IHostnameVerifier to be used for this HTTPS connection</returns>
+		/// <param name="connection">HTTPS connection object.</param>
+		protected virtual IHostnameVerifier GetSSLHostnameVerifier (HttpsURLConnection connection)
```

Comment:
Naming conventions: Should this really be `SSL` or should it be `Ssl`?

I'm also not sure if it should be `Hostname` or `HostName`, though given that the (pre-existing?) type is `IHostnameVerifier`, `Hostname` should probably be preferred.